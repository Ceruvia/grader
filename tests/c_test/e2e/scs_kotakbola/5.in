354 376 609
B 76
H 70
H 58
B 55
M 13
H 27
H 54
M 83
B 16
B 75
B 26
B 92
M 96
B 32
M 8
B 85
B 46
M 39
H 7
B 26
B 6
H 78
M 22
H 22
H 57
E