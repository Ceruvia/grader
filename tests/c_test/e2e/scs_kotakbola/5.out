373 261 535
Tidak
